external/upstream/libint2/CMakeLists.txt

```diff
@@ -46,6 +46,34 @@ find_package(
      eri_c4_d2_l2              eri_c3_d2_l3 eri_c2_d2_l3
   )
 
+# If L2 v2.8.0 (1) exported CMake components and (2) libint2-config.cmake actually
+#   checked, the below is the new find_package call.
+#if (${MAX_AM_ERI} GREATER 8)
+#    set(amchar "llll")
+#elseif (${MAX_AM_ERI} EQUAL 7)
+#    set(amchar "kkkk")
+#elseif (${MAX_AM_ERI} EQUAL 6)
+#    set(amchar "iiii")
+#elseif (${MAX_AM_ERI} EQUAL 5)
+#    set(amchar "hhhh")
+#elseif (${MAX_AM_ERI} LESS_EQUAL 4)
+#    set(amchar "gggg")
+#endif()
+#find_package(
```

Comment:
Would this have to be manually commented out by the user if they wanted to use Libint 2.8.0 instead of 2.7.3?